### ISSUE TITLE
[Feature]: Installation + instructions for x86 Linux hosts (Ubuntu, etc.)

### ISSUE THREAD

**Issue by CharlesCongdon** [hickey/meshchat#39]
### Contact Details

ccongdon42@gmail.com

### Enhancement Type

Installation method

### What is your idea or what can be improved?

I'd like to run MeshChat on an x86 Linux host.  I know I can do this with the old 1.0.2 Python version, but I'm not sure how to get the most recent LUA version up and running (2.10++) - I recall reading something on the arednmesh forums about LUA dependencies complicating this for Linux installations.   I'm also resisting installing the 1.0.2 Python version in case there are new features or necessary incompatibilities introduced in the new versions.
Thanks for all you do!
Charles

**Comment by hickey** [comment]
This is a duplicate of #6.

I am trying to finalize the release pipelines to get the first release as the new maintainer. Once that is done and I can get a valid release done, there will be focus put on the code to remove the Luci dependencies. This should allow MeshChat to run on a Linux OS.
